graspologic/utils/utils.py

```diff
@@ -701,3 +701,79 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remove_vertices(graph, indices, return_vertices=False):
```

Comment:
the name `return_vertices` doesn't seem quite right, it's kinda like returning a subgraph (or two)? not sure what to call it

Reply:
`return_removed`?

Reply:
sknetwork calls them "[adjacency_vectors](https://github.com/sknetwork-team/scikit-network/blob/master/sknetwork/embedding/spectral.py#L299)", we could use that

Reply:
i believe ["incidence matrix"](https://en.wikipedia.org/wiki/Incidence_matrix) is the correct term though I'm not necessarily advocating for it

Reply:
or I also think of them as subgraphs. did they have a name in the paper?

Reply:
return removed is fine

Reply:
9096bab